build/Publish.targets

```diff
@@ -43,16 +46,16 @@
       <ForPublishing Include="%(GenerateArchivesInputsOutputs.Outputs)"
                      Condition=" '$(PublishArchives)' == 'true' "/>
       <ForPublishing Include="$(PackagesDirectory)/Microsoft*.nupkg"
-                     Condition=" '$(PUBLISH_NUPKG_TO_AZURE_BLOB)' == 'true' AND '$(OS)' == 'Windows_NT' And '$(Architecture)' == 'x64' "/>
```

Comment:
Why did we decide to drop this flag? I like the ability to turn this off in draft or test build definitions for instance. I would like to keep the ability to run a VSO build that does not produce artifacts, for testing purposes, for instance.

Reply:
Maybe we change this to DO_NOT_PUBLISH_NUPKG_TO_AZURE_BLOB?

Reply:
We still have 'PUBLISH_TO_AZURE_BLOB'; that is the switch that controls the publish to blob storage. 'PUBLISH_NUPKG_TO_AZURE_BLOB' controls whether or not we publish the NuPkg for back-up to blob storage. Seems too granular to me.

Reply:
Ok. That works for me.